march_rqt_gait_generator/src/march_rqt_gait_generator/side_subgait_view.py

```diff
@@ -0,0 +1,19 @@
+from python_qt_binding.QtWidgets import (QCheckBox, QPushButton)
+
+
+class SideSubgaitView(object):
+    def __init__(self, widget):
+        self.import_button = widget.findChildren(QPushButton)[0]
+        self.default_checkbox = widget.findChildren(QCheckBox)[0]
+        self.lock_checkbox = widget.findChildren(QCheckBox)[1]
```

Comment:
Couldn't you use names here?

Reply:
Hmm not easily, because I call this same method for both side subgaits, and their names differ. I do now realise that i could pass a string `side` which can be either 'previous' or 'next' and use that to call by name anyway. Do you think that would be better?

Reply:
No you can leave it :+1: